cms/methodology/models.py

```diff
@@ -0,0 +1,95 @@
+from typing import TYPE_CHECKING, Any, ClassVar
+
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.functional import cached_property
+from django.utils.translation import gettext_lazy as _
+from wagtail.admin.panels import FieldPanel, FieldRowPanel, InlinePanel, MultiFieldPanel
+from wagtail.fields import RichTextField
+from wagtail.search import index
+
+from cms.core.blocks.stream_blocks import SectionStoryBlock
+from cms.core.fields import StreamField
+from cms.core.models import BasePage
+
+if TYPE_CHECKING:
+    from django.http import HttpRequest
+    from wagtail.admin.panels import Panel
+
+
+class MethodologyPage(BasePage):  # type: ignore[django-manager-missing]
+    parent_page_types: ClassVar[list[str]] = ["topics.TopicPage"]
+
+    template = "templates/pages/methodology_page.html"
+
+    summary = RichTextField(features=settings.RICH_TEXT_BASIC)
+    publication_date = models.DateField()
+    last_revised_date = models.DateField(blank=True, null=True)
+
+    contact_details = models.ForeignKey(
+        "core.ContactDetails",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    content = StreamField(SectionStoryBlock())
+
+    show_cite_this_page = models.BooleanField(default=True)
+
+    content_panels: ClassVar[list["Panel"]] = [
+        *BasePage.content_panels,
+        FieldPanel("summary"),
+        MultiFieldPanel(
+            [
+                FieldRowPanel(
+                    [
+                        FieldPanel("publication_date", icon="calendar-date"),
+                        FieldPanel(
+                            "last_revised_date",
+                        ),
```

Comment:
nitpick:

```suggestion
                        FieldPanel("last_revised_date"),
```

Reply:
Formatted that in [5690c11](https://github.com/ONSdigital/dis-wagtail/pull/57/commits/5690c1180196c129db181b75ac514dfbeac760dc).

I've also removed the `icon` for the `publication_date` because if I understand it correctly the icon displayed will be the one defined for the wrapping `MultiFieldPanel`, so the one here isn't necessary. 
I've done the same for the Analysis page for consistency.